### ISSUE TITLE
Versão de Java

### ISSUE THREAD

**Issue by hsven** [premium-minds/summer-internship-exercise-2022#2]
Boa tarde.
Notei que o ficheiro `pom.xml` tem a versão de Java configurada para Java 7. Esta versão é um requisito obrigatório do exercício ou temos a liberdade de usar uma versão diferente?

Obrigado.

**Comment by amadoandre** [comment]
Sim, tem liberdade de escolher a versão sendo sempre superior ou igual a 7
